### PR TITLE
Adapt to post-osgearth2.9 API changes in EarthManipulator

### PR DESCRIPTION
**Pull request by gwaldron**
Removed deprecated setTetherNode/getTetherNode functions from EM.

### CODE REVIEW

**Comment by gwaldron** [comment]
sorry let me resolve the conflict, standby

**Comment by gwaldron** [comment]
done!

File: SDK/simVis/View.cpp [hunk 1]
```diff
@@ -1556,10 +1562,11 @@ void View::enableWatchMode(osg::Node* watched, osg::Node* watcher)
         simVis::EarthManipulator* manip = dynamic_cast<simVis::EarthManipulator*>(getCameraManipulator());
         if (manip && manip->isTethering())
         {
+          osg::ref_ptr<osg::Node> tetherNode = manip->getViewpoint().getNode();
```
Comment (emminizer) [inline SDK/simVis/View.cpp]: No matching function call to osgEarth::Viewpoint::getNode()
Candidate expects 1 argument, 0 provided

**Comment by gwaldron** [comment]
This PR contains updates for osgEarth post-2.9. You need the latest master.

**Comment by emminizer** [comment]
We try to support back at least a little bit.  Like how osgEarth supports OSG 3.4.x and 3.5.x master.  We (try to) do the same.

File: SDK/simVis/View.cpp (hunk 1 above)
Comment (gwaldron) [reply SDK/simVis/View.cpp]: added ifdef

**Pull request merged**